### DISCUSSION TITLE
part-ii/2021-25/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#360] [Question comments]
Part II 2021 Automata and Formal Languages: Paper 3, Section I, F 

https://questions.tripos.org/part-ii/2021-25/

**Comment by artie2000** [comment]
All bookwork except for the countable case.

<details>
<summary>Countable case:</summary>

No in both cases.

Any finite language is regular, so take (for example) $L_i=\{p_i\}$, the $p^\text{th}$ prime, for a counterexample.

Since the complement of a regular language is regular, we can turn the last counterexample into a counterexample for the countable intersection.

</details>
